javascript/javascript3/week3/repositories_promise.js

```diff
@@ -0,0 +1,26 @@
+"use strict"
+
+const bodyTag = document.querySelector("body");
+
+
+const url = `https://api.github.com/search/repositories?q=user:`;
+
+const names = ['1620prabn', 'deepti-dk', 'yaser-aldammad']
+
+const promises = [];
+
+const fetchData = (name) => {
+    promises.push(fetch(`${url}${name}`))
+}
+
+names.map(fetchData)
+console.log(promises)
+
+const myClassmatesApi = () => {
```

Comment:
Good :)
You havent called myClassmatesApi() yet though,
Also try to think about the names you choose. 
A function should be a verb. 
data is not a descriptive name